src/transformers/models/bridgetower/image_processing_bridgetower.py

```diff
@@ -184,12 +205,15 @@ def __init__(
         do_center_crop: bool = True,
         crop_size: Dict[str, int] = None,
         do_pad: bool = True,
+        pad_and_return_pixel_mask: Optional[bool] = None,
+        return_tensors: Optional[bool] = None,
+        data_format: Optional["ChannelDimension"] = "channels_first",  # noqa: F821
+        input_data_format: Optional[Union[str, "ChannelDimension"]] = None,  # noqa: F821
         **kwargs,
     ) -> None:
-        if "pad_and_return_pixel_mask" in kwargs:
-            do_pad = kwargs.pop("pad_and_return_pixel_mask")
+        if pad_and_return_pixel_mask:
+            do_pad = pad_and_return_pixel_mask
```

Comment:
I think I let this slip through the net when I was reviewing. If we're doing this, then we should really have a deprecation message to 1) avoid this dual arg use 2) remove x_and_y kind of arguments

Reply:
I think before it was just popping from kwargs the x_and_y arg and assigning to `do_pad`. I'm all for removing dual args in general, and deprecation warnings are indeed missing from that one - will add that!